### PR TITLE
fix: converting list indents(wysiwyg => markdown)

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [ x It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* apply the same indent to list item when converting wysiwyg to markdown.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/editor/src/convertors/toMarkdown/toMdNodeTypeWriters.ts [hunk 1]
```diff
@@ -125,18 +125,18 @@ export const nodeTypeWriters: ToMdNodeTypeWriterMap = {
   },
 
   bulletList(state, { node }, { delim }) {
-    state.convertList(node, '  ', () => `${delim} `);
+    // soft-tab(4)
+    state.convertList(node, '    ', () => `${delim} `);
```
Comment (jung-han) [inline apps/editor/src/convertors/toMarkdown/toMdNodeTypeWriters.ts:129]: 스페이스로 되어있는 문자열 대신에 숫자를 넣으면 반환하는 함수로 나타내고 주석을 없애는건 어떤가요?

**Review by jung-han — APPROVED** [review]
완료입니다!

**Review by jungeun-cho — APPROVED** [review]
이견없습니다.

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Pull request merged**